utils/common/src/test/kotlin/EnvironmentVariableFilterTest.kt

```diff
@@ -80,4 +80,18 @@ class EnvironmentVariableFilterTest : StringSpec({
 
         environment should containExactly(filteredEnvironment)
     }
+
+    "An empty deny list should not deny anything" {
+        EnvironmentVariableFilter.reset(denySubstrings = emptyList())
+
+        EnvironmentVariableFilter.DEFAULT_DENY_SUBSTRINGS.forAll {
+            EnvironmentVariableFilter.isAllowed(it) shouldBe true
+        }
+
+        EnvironmentVariableFilter.DEFAULT_ALLOW_NAMES.forAll {
+            EnvironmentVariableFilter.isAllowed(it) shouldBe true
+        }
+
+        EnvironmentVariableFilter.isAllowed("PATH") shouldBe true
```

Comment:
Maybe here it could make sense to use a variable which doesn't exist in reality, so one does not need to think about whether it's part of default allow / deny collection?! e.g. `SOME_VARIABLE

Reply:
I had chosen `PATH` as that's specifically what https://github.com/oss-review-toolkit/ort/issues/9604 says to be filtered out.